templates/source/sourceExample.md

```diff
@@ -0,0 +1,96 @@
+# Event source for AWS Cognito User Pool
+
+This event source captures messages from an [AWS Cognito User Pool ][cup-docs] whenever a specific action, such as the creation of a new user, happens in the user identity pool.
+
+## Prerequisites
+
+- AWS Cognito User Pool
+- Amazon Resource Name (ARN)
+- API Credentials
+
```

Comment:
Maybe not an issue, but I find it hard and ugly the lack of indentation between ## and #####.
I think that is something that can be worked on the `mkdocs.yml` at the root. Should we create a followup to make this look nicer?

![image](https://user-images.githubusercontent.com/9891289/110340890-0d985180-802a-11eb-8c30-1765d81079bb.png)

Reply:
Are you wanting this to be indented only are there other styles about it that you didn't like?

Reply:
IMHO maybe adding weight - bolding the ## is enough.
It is still a bit subtle that transition between ## and ##### when prerequisites take more than one scroll page.

Reply:
Yeah, just when it was ### there was nothing differing the prerequisites with the sub information. THe API Credentials and ARN are supposed to be subheading of prerequisites. The main title is #. The main information is ##. And the sub info would be #####. It is just a part of the prerequisites so I needed to make the titles different.